app/controllers/operations_controller.rb

```diff
@@ -0,0 +1,86 @@
+class OperationsController < ApplicationController
+  before_action :set_operation, only: %i[show edit update destroy new create]
+  before_action :authenticate_user!
+
+  def index
+    @group = Group.find(params[:group_id])
+    @operations = @group.operations.order(created_at: :desc)
+  end
+
+  # GET /operations/1 or /operations/1.json
+  def show; end
+
+  # GET /operations/new
+  def new
+    @operation = Operation.new
+    @operations = Operation.all
+    @groups = Group.created_by_current_user(current_user)
+  end
+
+  # GET /operations/1/edit
+  def edit; end
+
+  # POST /operations or /operations.json
+  # @group = Group.operations.find(params[:id])
```

Comment:
- [ ] Please remove commented-out code as it is of no use, makes your code look cluttered, and takes up space
- [ ] Kindly do the same for the groups_controller and splash_controller